doc/sphinx/source/community/release_strategy.rst

```diff
@@ -330,6 +331,9 @@ git using [setuptools-scm](https://pypi.org/project/setuptools-scm/), but a
 static version number is stored in ``CITATION.cff``.
 Make sure to update the version number and release date in ``CITATION.cff``.
 See https://semver.org for more information on choosing a version number.
+Update the `environment.yml <https://github.com/ESMValGroup/ESMValTool/blob/main/environment.yml#L25>`__
```

Comment:
It seems that setup.py is missing here, but maybe you could just link to the section on [updating dependencies](https://esmvaltool--2734.org.readthedocs.build/en/2734/community/code_documentation.html#dependencies) to avoid duplicating information? That would also help to keep the information up to date if things change again in the future.

Reply:
Changed

Reply:
Thanks